vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/Status.kt

```diff
@@ -0,0 +1,12 @@
+package at.asitplus.wallet.lib.data
+
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.StatusListInfo
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.status.JwtStatusListStatusMechanismSpecification
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Status(
+    @SerialName(JwtStatusListStatusMechanismSpecification.NAME)
```

Comment:
Why do we need the `Cwt*` classes then, if here the `Jwt*` are used?

Reply:
removed status mechanisms and status claim since there is currently no use for them